modules/gapi/src/api/gmat.cpp

```diff
@@ -48,10 +48,16 @@ namespace{
     }
 }
 
+cv::GMatDesc cv::gapi::own::descr_of(const Mat &mat)
```

Comment:
It duplicates the code from `cv::GMatDesc cv::descr_of(const cv::Mat &mat)`. 
But we can call `cv::descr_of(...)` for Mat (STANDALONE / OCV) or call `cv::gapi::own::descr_of(...)` for cv::gapi::own::Mat (this is used only in the test and doesn't depend on the flag `GAPI_STANDALONE`).